paws_collector.js

```diff
@@ -168,9 +168,11 @@ class PawsCollector extends AlAwsCollector {
     };
 
     done(error, pawsState, sendStatus = true) {
+        // If stream exist in state then post the error status as part of stream specific ; 
+        // In check-in or self-update we don't get state ,so check collector_streams env variable and post error as part of paws_{applicationId}_status else post the error paws_logmsgs_status
         const streamType = pawsState && pawsState.priv_collector_state.stream ?
             process.env.al_application_id + "_" + pawsState.priv_collector_state.stream :
-            null;
+            process.env.collector_streams ? process.env.al_application_id : null;
```

Comment:
I think we just want to have `process.env.al_application_id` regardless of `process.env.collector_streams` value.